src/pages/review/Content.js

```diff
@@ -1,24 +1,32 @@
 import React from 'react';
+import { useNavigate } from 'react-router-dom';
 import styled from 'styled-components';
 
-const Content = () => {
+const Content = ({ reviewList }) => {
+  const navigate = useNavigate();
+
+  const goToDetail = () => {
+    navigate(`/review/detail/${reviewList.id}`);
+  };
+
   return (
     <>
       <Top>
         <Preview>
-          <span>박**</span> (오메가3, 비타민B, 밀크씨슬)
+          <span>{reviewList && reviewList.name.slice(0, 1)}**</span>
+          (오메가3, 비타민B, 밀크씨슬)
         </Preview>
-        <Period>2022.05.27 / 정기구독 18개월</Period>
+        <Period>
+          {reviewList && reviewList.date} / 정기구독{' '}
```

Comment:
```suggestion
          {reviewList?.date} / 정기구독{' '}
```
&&를 이용한 단축평가와 동일한 동작을 optional chaining 문법 이용해서 좀 더 간결하게 표현할 수도 있습니다. [공식문서](https://developer.mozilla.org/ko/docs/Web/JavaScript/Reference/Operators/Optional_chaining) 참고해서 어떻게 동작하는지 확인 및 이해해보시고 적용해보세요!